spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -187,6 +187,9 @@ def __init__(
         self.is_kernel_configured = False
         self._init_kernel_setup = False
         self._is_banner_shown = False
+        # Set bright colors instead of bold formating for better traceback
+        # readability
+        self._ansi_processor.bold_text_enabled = False
```

Comment:
```suggestion

        # Set bright colors instead of bold formatting for better traceback
        # readability.
        self._ansi_processor.bold_text_enabled = False
```

Improve comment a bit.